### PR TITLE
Bug fixes for Expand PVC

### PR DESCRIPTION
**Pull request by cloudbehl**
github-issue: https://github.com/openshift/origin-web-console/issues/3039

### CODE REVIEW

File: app/scripts/controllers/persistentVolumeClaim.js [hunk 1]
```diff
@@ -38,11 +38,16 @@ angular.module('openshiftConsole')
     var storageClass = $filter('storageClass');
     var watches = [];
 
+    var scResolved = function (sc) {
+      $scope.isExpansionAllowed = (!sc || sc.allowVolumeExpansion) && $scope.pvc.status.phase === 'Bound';
```
Comment (spadgett) [inline app/scripts/controllers/persistentVolumeClaim.js]: 👍

**Comment by spadgett** [comment]
Test failure is a cluster up flake

/retest

**Comment by spadgett** [comment]
@deads2k Is the test failure a cluster up bug?

```
+ oc auth reconcile --config=/kubeconfig.kubeconfig -f -
error: only rbac.authorization.k8s.io/v1 is supported: not *v1beta1.ClusterRole
I0723 12:21:22.016680    3044 run.go:293] Deleting container "20519334a6a09802644e294567e8ce29964a5b2f11edd96526cf8195894dcd3d"
E0723 12:21:22.032506    3044 interface.go:34] Failed to install "openshift-controller-manager": failed to install "openshift-controller-manager": Docker run error rc=1; caused by: Docker run error rc=1
I0723 12:21:22.032539    3044 interface.go:41] Finished installing "openshift-controller-manager"
Error: failed to install "openshift-controller-manager": Docker run error rc=1; caused by: Docker run error rc=1
```

https://openshift-gce-devel.appspot.com/build/origin-ci-test/pr-logs/pull/openshift_origin-web-console/3040/test_pull_request_origin_web_console/2392/

**Comment by deads2k** [comment]
> @deads2k Is the test failure a cluster up bug?

It indicates that the content being passed includes a `v1beta1` version of RBAC resources and those won't work with `oc auth reconcile`.  You'll need to find and update the offending RBAC resources.

**Comment by spadgett** [comment]
@deads2k But this is a fresh cluster up with no previous data

**Comment by cloudbehl** [comment]
@spadgett Removed WIP patch! Thanks

**Comment by spadgett** [comment]
Debugged this with @deads2k. cluster up is failing because origin:latest image is a month old, so we're getting an old version of oc

**Comment by cloudbehl** [comment]
@spadgett Is there anything that I need to look at, for the latest image issue?

**Comment by spadgett** [comment]
@cloudbehl No, our CI for all origin-web-console PRs is broken. We're looking at it.

**Comment by deads2k** [comment]
> Debugged this with @deads2k. cluster up is failing because origin:latest image is a month old, so we're getting an old version of oc

I looked it up.  I think we're now using openshift/origin-cli

**Comment by spadgett** [comment]
CI fix in https://github.com/openshift/aos-cd-jobs/pull/1532

Thanks @deads2k

**Pull request merged**